### PR TITLE
remove unused `MaybeUninit`

### LINKED ISSUE #15: warning: unused `use core::mem::MaybeUninit;`
The compiler warns: "unused import: `core::mem::MaybeUninit`". What is this module supposed to be for? Can it be deactived by default, to not confuse beginners like me?

### PR DESCRIPTION
**Pull request by pkoevesdi**
fixes #15

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #![no_std]
 #![no_main]
 
-use core::mem::MaybeUninit;
+// use core::mem::MaybeUninit;
```
Comment (Yatekii) [inline src/main.rs]: We can just remove the line of code :)

Comment (pkoevesdi) [reply src/main.rs]: Great, thanks!

Comment (Yatekii) [reply src/main.rs]: I believe the comment is still there instead of removing the line :P

Comment (pkoevesdi) [reply src/main.rs]: Ups, sorry, I thought by resolving, it takes the suggestion. I make new commit later.

Comment (Yatekii) [inline src/main.rs]:
```suggestion
```

Comment (Yatekii) [reply src/main.rs]: You can accept this suggestion, that works :)

**Review by Yatekii — APPROVED** [review]

**Pull request merged**